### DISCUSSION TITLE
Can we add http related metrics in kong datadog plugin as only default is coming

### DISCUSSION THREAD

**Discussion by bikasit** [Kong/kong#11358] [Help]
We have integrate datadog plugin in kong but only able to see  or fetch kong.default-plugin.* not able to see other related metrics

like , kong.http.<service-name>.status.count, etc

can anybody help

**Comment by samugi** [comment]
Hi @bikasit once the plugin is correctly configured pointing to the correct host and port of your Datadog agent it should be able to communicate and send all metrics, which can be filtered by name from the Datadog metrics explorer, e.g. `kong.response.*`.
Please make sure your agent is reachable by Kong and configured to accept statsd traffic (even non local if appropriate).

Also, if you are interested in expanded tracing data and functionalities please be aware that it is possible to configure your Datadog agent to [accept OTLP ingestion](https://docs.datadoghq.com/opentelemetry/otlp_ingest_in_the_agent/?tab=host) and then use the [Kong Opentelemetry](https://docs.konghq.com/hub/kong-inc/opentelemetry/) plugin to communicate with Datadog.
